CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
   As this change is backward compatible in Cumulus Core, users wishing to utilize the previous version of the CMA may opt to transition to using a CMA lambda layer, or set `message_adapter_version` in their configuration to a version prior to v1.1.0.
 
+- **CUMULUS-1449** - Workflow configurations for the `QueueGranules` and `QueuePdrs` tasks need to be updated.
```

Comment:
Are there going to be migration instructions for this?

Reply:
Updated in 44e806a58